DSCResources/MSFT_xKDSKey/MSFT_xKDSKey.meta.mof

```diff
@@ -0,0 +1,5 @@
+[ClassVersion("1.0.0.0"), FriendlyName("xKDSKey")]
```

Comment:
Please document the change in the unreleased section in https://github.com/PowerShell/xActiveDirectory/blob/dev/README.md